api/models/loomies.go

```diff
@@ -250,3 +250,27 @@ func InsertUserInArrayOfWildLoomie(loomie interfaces.WildLoomie, user interfaces
 
 	return err
 }
+
+func UpdateLevelOfLoomie(userId primitive.ObjectID, loomieId primitive.ObjectID) (bool, error) {
```

Comment:
I suggest to update this function to make it "more general": You can also receive the amount of levels the Loomie will increase.